hibp.js

```diff
@@ -29,14 +31,14 @@ const HIBP = {
       response = await got(url, reqOptions);
       return response;
     } catch (err) {
-      console.error("got an error: " + err);
+      log.error("_throttledGot", {err: err});
       if (err.statusCode === 404) {
         // 404 can mean "no results", return undefined response; sorry calling code
         return response;
       } else if (err.statusCode === 429) {
-        console.log("got a 429, tryCount: ", tryCount);
+        log.info("_throttledGot", {err: "got a 429, tryCount: " + tryCount});
         if (tryCount >= AppConstants.HIBP_THROTTLE_MAX_TRIES) {
-          console.error(err.message);
+          log.error("_throttledGot", {err: err});
```

Comment:
I'm not 100% familiar w/ mozlog internals, but I'm noticing that sometimes it looks like we're logging an object vs possibly nested error objects (if that matters, and maybe it doesn't):

Above (L39) we do `log.info("_throttledGot", {err: "some string"})`, but here, we're logging an Error object instance, which may be something closer to;

```js
log.error("_throttledGot", {err: {message: "some error message", "stack": "blah"}});
```

So in one branch its `err:string` and in other branches it's `err:object`.
Not saying it's wrong or bad, just invoking "see-something-say-something".